lib/runner.js

```diff
@@ -52,11 +56,29 @@ class Runner extends Emittery {
 			if (!scheduledStart) {
 				scheduledStart = true;
 				process.nextTick(() => {
+					this.callsiteLineNumbers.push(Infinity);
+					this.callsiteLineNumbers.sort((a, b) => a - b);
 					hasStarted = true;
 					this.start();
 				});
 			}
 
+			if (this.lines.length > 0) {
+				let callsiteLineNumber;
+
+				for (const jsCallsite of callsites()) {
+					const callsite = sourceMapSupport.wrapCallSite(jsCallsite);
+					const fileName = callsite.getFileName();
+					if (jsCallsite === this.file || fileName === this.file) {
```

Comment:
Did you mean `jsCallsite.getFileName()`?

I don't think this needs to be compared to the source-map converted filename. You can't pass those on the CLI since you need to provide the file that AVA can actually load.